libr/bin/p/bin_dex.c

```diff
@@ -721,6 +721,13 @@ static void *load_bytes(RBinFile *bf, const ut8 *buf, ut64 sz, ut64 loadaddr, Sd
 	return res;
 }
 
+static void * load_buffer(RBinFile *bf, RBuffer *buf, ut64 loadaddr, Sdb *sdb) {
+	if (!buf) {
```

Comment:
we may probably want to move this check into r_bin_dex_new_buf() to avoid a crash when using the api in a wrong way